plugins/modules/docker_container_exec.py

```diff
@@ -163,8 +163,13 @@ def main():
         tty=dict(type='bool', default=False),
     )
 
+    option_minimal_versions = dict(
+        chdir=dict(docker_py_version='3.0.0'),
+    )
```

Comment:
holy ravioli, I hadn't seen this construct before, is it new? awesome :-)

Reply:
No, it's actually pretty old ;-) I originally created it for docker_container, since it has sooo many options and these have so many different requirements...

Reply:
Cool, will take a look at it!